Remotion/ObjectBinding/Web/Doc/include/UI/Controls/BocReferenceValue.xml

```diff
@@ -44,10 +44,6 @@
         <item>
           <term>WAI WCAG 1.0</term>
           <description>
-            <para> 
-              Level A conform if advanced features are turned off. This can be enforced by the 
-              <see cref="Remotion.Web.Configuration.WebConfiguration.Wcag"/> configuration entry.
-            </para>
```

Comment:
I wouldn't delete the whole paragraph here. I would leave `Level A conform if advanced features are turned off.` since we only removed the enforcement and add this spot to the list of TODOs just like the other two places that have open XML doc questions